docs/paper/dev/api/inventory-holder.md

```diff
@@ -0,0 +1,156 @@
+---
+slug: /dev/inventory-holder
+---
+
+# InventoryHolder
+
+Using an InventoryHolder is a great way of identifying your plugin's inventories in events.
+
+## Why use an InventoryHolder?
+
+InventoryHolder greatly simplifies the steps you need to do to make sure an inventory was created by your plugin.
+
+Using inventory names for identification is unreliable as other plugins can create inventories with names exactly as yours
+and with Components you need to make sure the name is exactly the same or serialize it to other formats.
```

Comment:
This should be emphasized more. maybe some kind of colored box like note or info? Idk all the available ones.